src/OrleansAzureUtils/Hosting/AzureSilo.cs

```diff
@@ -154,21 +155,20 @@ public bool Start(ClusterConfiguration config, string deploymentId = null, strin
             // Read endpoint info for this instance from Azure config
             string instanceName = serviceRuntimeWrapper.InstanceName;
 
+            var builder = new SiloHostBuilder();
+
             // Configure this Orleans silo instance
             if (config == null)
             {
-                host = new SiloHost(instanceName);
-                host.LoadOrleansConfig(); // Load config from file + Initializes logger configurations
-            }
-            else
-            {
-                host = new SiloHost(instanceName, config); // Use supplied config data + Initializes logger configurations
+                builder.UseClusterConfiguration(config); // Use supplied config data + Initializes logger configurations
             }
 
+            builder.ConfigureSilo(silo => 
```

Comment:
This mega lambda exists only because I didn't think of how to do all of what this was doing before the "initialize" was called.  This would definitely be cleaned up if this general direction is liked.